colcon_bundle/verb/_overlay_utilities.py

```diff
@@ -81,3 +130,30 @@ def recursive_tar_gz_in_path(output_path, path):
         for name in os.listdir(path):
             some_path = os.path.join(path, name)
             tar.add(some_path, arcname=os.path.basename(some_path))
+
+
+def generate_template(template_name, script_name, context_vars):
```

Comment:
```suggestion
def _generate_template(template_name, script_name, context_vars):
```

let's make private functions private